docs/features/nullable-metadata.md

```diff
@@ -0,0 +1,191 @@
+Nullable Metadata
```

Comment:
nit: FWIW, I'd prefer if we merged this doc into the main design doc.

Reply:
I'd prefer to see this as a separate document because the metadata spec is now longer and can stand alone.

---
In reply to: [293550340](https://github.com/dotnet/roslyn/pull/36398#discussion_r293550340) [](ancestors = 293550340)